nixos/modules/services/web-apps/miniflux.nix

```
@@ -13,6 +13,7 @@ let
   preStart = pkgs.writeScript "miniflux-pre-start" ''
     #!${pkgs.runtimeShell}
     ${pgbin}/psql "${dbName}" -c "CREATE EXTENSION IF NOT EXISTS hstore"
+    ${pgbin}/psql "${dbName}" -c "ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT ALL PRIVILEGES ON TABLES TO ${dbUser};"
```

```suggestion
    ${pgbin}/psql "${dbName}" -c "ALTER DATABASE ${dbName} OWNER TO ${dbUser};"
```

Taken from #262741 (it was Forgejo, not Gitea)